### DISCUSSION TITLE
Literally How to Play This Game?

### DISCUSSION THREAD

**Discussion by zina0316** [meikpiep/gauguin#171] [Q&A] [answered]
Is there any tutorial thing or something?
because I just can't solve the puzzle more than easy

**Comment by meikpiep** [comment]
Actually not. I did not find a way to teach the game which is self-explanatory. May be, an interactive tutorial will suffice.

I may point you to a video from Cambridge Nrich: https://nrich.maths.org/15109 from this website https://nrich.maths.org/15108 - the content is targeting pupils, but the video is rather good in my opinion. The principle is identical, though in Gauguin, you can add pencil marks short-tapping a number and enter a solutions number by long-tapping a number.

Are you struggling with the rules or with some solving techniques?

I would be glad if we get to a better onboarding process.

**Comment by zina0316** [comment]
Oh I didn't know it's called **Mathdoku**
and I kept searching **KenKen**
I'll watch those tutorials later when I have time

but I still have a TINY question (I don't want to open another discussion to talk about it) 
How does **Fast Finishing Mode** work?
Like, how does it define **At the end of the game**?
I just can't input with taps even there's only one possibility to the cell

**Reply by meikpiep** [reply, marked as answer]
You're welcome!

KenKen is the official name, but there is a company which gains copyright on it, so many games (like Gauguin) avoid the term and use MathDoku instead.

The Fast Finishing Mode gets activated if you fill subsequent cells via long tapping on cells with single pencils marks. Any other action like entering the number via the number button avoids the mode. Hope this helps.

**Comment by zina0316** [comment]
Hey, Do you know any discord channel or sth like that to discuss mathdoku?
I often stuck on some puzzle and don't know the proper way to solve that (I just guess most of the time)

Like this one, I don't know where to start solve it
![Screenshot_20240605_005335_Gauguin](https://github.com/meikpiep/gauguin/assets/94322065/0c228f29-5917-4be9-9890-a0d2c841e08e)

**Reply by meikpiep** [reply]
Unfortunately no. But I can describe how I would solve it. Please give feedback if I'm too detailed or to fast when explaining, I'm just guessing what would help.

On the left side, there is a '6+' in a straight line. The only possibility of numbers which fullfil this is 1,2,3 adding to 6 (each column contains exactly the values 1,2,3,4 in a variable order, so we can't use a value more than once if the cage contains cells of only one row or column. We don't know in which cell to fill these numbers in the cage, but anything containing a 4 would be bigger than 6: Combining 4 with the lowest numbers 1 and 2 gives 7.

Thus, you could eliminate the 4 from any cell of the '6+'. This leads to put the 4 into the lower left corner as no other cell of this column could contain it.

On the right side, you could deal with the '8+' by either trying all combinations which add to 8: There is only 1+3+4=8 in a column. So you can put the 2 into the cell below. To explain another technique, you can sum up all numbers in a row or column, which are always the same values: In 4x4, there are 1+2+3+4=10 - each row and each column add to exactly 10. Thus, you could calculate the value of the lower right cell as 10 - 8 = 2, as the remaining cells all belong to the cage '8+' which shows us it sum straight away.

**Reply by zina0316** [reply]
Thank you very much
but now I have another puzzle to ask 😅
![Screenshot_20240605_222009_Gauguin](https://github.com/meikpiep/gauguin/assets/94322065/256942ab-c83f-4400-9026-9450c3f1573c)

**Reply by meikpiep** [reply]
In the lowest row, there is a '3-' with two cells. If you try to combine the three possible values which are left in the row (1,2,4), you can find that there is one single combination of that three fulfilling the diff of 3. The one value left can be put into the cell left of the 3 in the lower row.
